### DISCUSSION TITLE
Axios showing Inerita rendered data

### DISCUSSION THREAD

**Discussion by robov** [inertiajs/inertia#1205] [Help]
I want to realize the following

- Make axios call to the backend
- on errors return json erros
- frontend shows errors
- on no errors return interia render
- frontend shows new page

I am not sure if and how this can be realized. When I return a inertia render my axios then callback received the screen but how do I display this.

My current workaround is to place the backend call and if success do an inertia.get to get the new page, but the drawback of this is that I have 2 backend calls (post and get). 

When I do an inertia.post I get a redirect on validation errors by the backend, and I cannot postprocess in the frontend what the backend passed data.

Any suggestions ?

**Comment by taozuhong** [comment]
Any progress with it?
